### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by ek121268**
Basic documentation to run the dotnet demo

### CODE REVIEW

File: examples/dotnet/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+**Install the IDE Plugin**
+
+The first step is to install the Lightrun plugin to your IDE from the IDE marketplace extension menu. The instructions for this will vary based on the IDE you are using.
+
+You will then need to log in with the plugin.
+
+Install the .NET package by running the following command from your IDE terminal
+
+**dotnet add package Lightrun**
```
Comment (mjsambol) [inline examples/dotnet/README.md]: should be formatted as code

File: examples/dotnet/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+**Install the Lightrun Agent**
+
+The Lightrun agent (https://docs.lightrun.com/dotnet/agent-configuration/) should be added to your project together with your agent secret key that can be obtained from the Lightrun management portal (https://app.lightrun.com).
+
+In the Program.cs code sample the relevant code lines are already added, you should only specify your secret key.
+
+**Starting the Program**
+
+To run the program, simply open a terminal within your IDE, or command-lineand navigate to the folder which contains the Program.cs file, and execute the app.
```
Comment (mjsambol) [inline examples/dotnet/README.md]: command-lineand 
should probably be or access the command line however you would normally, navigate to the folder ...

File: examples/dotnet/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+**Adding Actions**
+
+You can now begin adding actions, such as Dynamic Logs (including conditional), and Snapshots (including conditional).
```
Comment (mjsambol) [inline examples/dotnet/README.md]: Now that the application is running you can begin adding Lightrun actions such as Dynamic Logs and Snapshots from your IDE.
By doing so you can observe the run-time state of the application without changing the code!

**Review by mjsambol — CHANGES_REQUESTED** [review]

**Review by ek121268 — COMMENTED** [review]
Readme modifications based on Moshe's review requests

**Review by mjsambol — APPROVED** [review]

**Pull request merged**